crates/core/src/otel.rs

```diff
@@ -32,6 +32,13 @@ pub struct OtelConfig {
     /// Determines whether http or grpc will be used for exporting the telemetry.
     #[serde(default)]
     pub protocol: OtelProtocol,
+    /// The level of tracing to enable.
+    #[serde(default = "default_trace_level")]
+    pub trace_level: Level,
+}
+
+fn default_trace_level() -> Level {
+    Level::Debug
```

Comment:
Do we want to default tracing to `debug` instead of `info`?

Reply:
Well mostly the note in #1932, and the levels that we instrument today generally have useful instrumentation at the debug level and only the minimum information at info level. We could have a broader discussion regarding our use of trace levels

Reply:
Put to INFO by default 😄